vendor/k8s.io/utils/internal/third_party/forked/golang/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012 The Go Authors. All rights reserved.
```

Comment:
I'm +1 for picking this, but it seems off that this LICENSE doesn't get pulled into the k/k licenses in any way... should k8s.io/utils update/include this somehow?

Reply:
@liggitt good point. just realized we don't do `third_party/forked/golang/LICENSE` in kubernetes/kubernetes either. will tackle both of them together. let's get this in and i'll file an issue/pr afterwards

Reply:
https://github.com/kubernetes/kubernetes/issues/103551

Reply:
PR is here : https://github.com/kubernetes/kubernetes/pull/103556